### PR TITLE
add more info about PR contribution

### PR DESCRIPTION
**Pull request by BoDonkey**
## Summary

*Summarize the changes briefly, including which issue/ticket this resolves. If it closes an existing Github issue, include "Closes #[issue number]"*
Adds more info about creating a PR for contributing to core or the documentation. This change is due to problems that a community member was experiencing. Closes PRO-3368
 
## What are the specific steps to test this change?

*For example:*
> 1. Run the website and log in as an admin
> 2. Open a piece manager modal and select several pieces
> 3. Click the "Archive" button on the top left of the manager and confirm that it should proceed
> 4. Check that all pieces have been archived properly

1. Open and read the `CONTRIBUTING.md` file.

## What kind of change does this PR introduce?
*(Check at least one)*

- [ ] Bug fix
- [ ] New feature
- [ ] Refactor
- [X] Documentation
- [ ] Build-related changes
- [ ] Other

## Make sure the PR fulfills these requirements:

- [X] It includes a) the existing issue ID being resolved, b) a convincing reason for adding this feature, or c) a clear description of the bug it resolves
- [ ] The changelog is updated
- [ ] Related documentation has been updated
- [ ] Related tests have been updated

If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -95,14 +98,34 @@ were applicable. For Apostrophe core, that should be in the
 and for other modules, add it in their README files (unless the README directs
 you elsewhere).
 
+### Contributing to Apostrophe Core
+
+This is assuming you are interacting with the ApostropheCMS repositories on the GitHub website. If you are using GitHub Desktop you can read about how to for a repository in the [GitHub docs.](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/adding-and-cloning-repositories/cloning-and-forking-repositories-from-github-desktop)
```
Comment (boutell) [inline CONTRIBUTING.md:103]: for -> fork

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -95,14 +98,34 @@ were applicable. For Apostrophe core, that should be in the
...
+3. Once you've completed your code changes (and updates to `CHANGELOG.md` if needed - see the notes above) you can push all of your changes to your repo. Navigating to your GitHub repository page you will see a banner for creating a PR. Click on the "Contribute" and then "Open a pull request" buttons.
+4. This will bring up a PR page. There are a number of sections to be filled out. Please read carefully and make selections where needed. Following this checklist is very helpful when our team reviews your request. If you need help, just ask!
+5. Finally, it is best to notify a team member on the [Discord channel](https://discord.com/channels/517772094482677790/701815369005924374) that you have submitted a PR. You can also find our contact addresses on our GitHub pages. We don't get automatic notifications from community-submitted PR's. We will see it in the queue eventually, notifying us will just speed up the process.
```
Comment (boutell) [inline CONTRIBUTING.md:109]: No apostrophe in "PRs" since nothing is being contracted

**Review by boutell — CHANGES_REQUESTED** [review]
Good idea. Minor nits.

File: CONTRIBUTING.md (hunk 1 above)
Comment (BoDonkey) [reply CONTRIBUTING.md:103]: FIxed.

File: CONTRIBUTING.md (hunk 2 above)
Comment (BoDonkey) [reply CONTRIBUTING.md:109]: Fixed.

**Pull request merged**